src/webrtc/getUserMedia.ts

```diff
@@ -5,9 +5,10 @@ export function getUserMedia(
     return navigator.mediaDevices.getUserMedia(constraints);
   }
 
-  if (navigator.getUserMedia) {
+  const nav: any = window.navigator;
+  if (nav.getUserMedia) {
```

Comment:
Same question, why `: any` ?
Could it be replaced with `if ('nativator' in window)` ?